lib/ip_access_control.ex

```diff
@@ -116,7 +116,8 @@ defmodule IpAccessControl do
 
   alias Plug.Conn
 
-  @typep ip_block_list :: [%RemoteIp.Block{}, ...]
+  # @typep ip_block_list :: BitwiseIp.Blocks.t()
```

Comment:
This would be ideal to be able to do where `BitwiseIp.Blocks.t :: list(BitwiseIp.Blocks.t)`.

Reply:
Makes sense. Pushed.